### DISCUSSION TITLE
Add to-do list

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#227] [General]
I would like a full list of features that are already added and are going to be added in the README.md or in a pinned issue. this would give people a look into what the project already has to offer and what we can expect to see in the future.

**Comment by DustinBrett** [comment]
I have attempted to list most/all existing features within the [README.md](https://github.com/DustinBrett/daedalOS/blob/main/README.md#system-), if I have missed any please mention them here and I will update that file. As for making a TODO.md, that is a good idea and I will likely do that very soon. I recently updated all my post-it's which I use to keep track of my plans for daedalOS. Although it should be noted I do this in my spare time and some of the features may never happen or could be decades off. But I hope to still be doing the occasional commit to this thing in 2050.

**Reply by DustinBrett** [reply]
Here is a picture of my current post it wall.

![20221028_192254](https://user-images.githubusercontent.com/16656318/198775059-193fa766-ffe8-4125-85ee-a6bfc7a30f6c.jpg)

**Reply by CorruptionVR** [reply]
hey, quick question. what does cyan mean? as in, the one in the middle of the board.

**Reply by DustinBrett** [reply]
Haha, good question. I think I'd decided it was things that I wanted to do asap, just to get it done. But the coloring when it comes to features is a bit confusing sometimes, even to me.

**Reply by CorruptionVR** [reply]
oh, and is there a difference for the features between orange and yellow, like is yellow a higher priority or something?

**Comment by CorruptionVR** [comment]
I have crossed out all of the post-its that I can read, but I cannot read the rest due to image quality. could you please send me closer images of the rest?
![ToDo-board](https://user-images.githubusercontent.com/78228404/199077365-e3420adc-1c97-466b-a3f6-50e4c1596d41.jpg)

**Comment by CorruptionVR** [comment]
here is the file that I have made for the todo.md
[Todo.md](https://github.com/DustinBrett/daedalOS/files/9912242/Todo.md)

**Comment by DustinBrett** [comment]
Thanks for creating that markdown document. I'm also playing with the idea of using Issues/Discussions more actively, as well as the Projects boards. I will look into a way to keep people more aware of these notes as their updates.
